core/common/src/main/java/alluxio/AlluxioURI.java

```diff
@@ -448,4 +449,18 @@ public String toString() {
     }
     return sb.toString();
   }
+
+  /**
+   * Gets the path component of the {@link AlluxioURI}.
+   *
+   * @return the absolute path
+   */
+  public String getAbsolutePath() {
+    if (mUri.getAuthority() != null) {
+      return PathUtils.concatPath(System.getProperty("user.dir"),
+              mUri.getAuthority().equals(".") ? mUri.getPath() : mUri.getAuthority());
```

Comment:
@aaudiber  `AlluxioURI` is fully based on the `alluxio.URI`. You mean, if we want to fix the `AlluxioURI`, we should completely rewrite the how `alluxio.URI` parse the  path?